### ISSUE TITLE
BroccoliMergeTrees: Assembler (vendor & appJS) failed with: no such file or directory

### ISSUE THREAD

**Issue by Turbo87** [kellyselden/ember-cli-tree-shaker#9]
In `ember-cli-build.js` I'm importing using:

```js
app.import('node_modules/animation-frame/AnimationFrame.js');
```

however the tree-shaker shows:

```
node_modules/animation-frame/AnimationFrame.js
node_modules/animation-frame/AnimationFrame.min.js
node_modules/animation-frame/FpsMeter.js
```

as being excluded from the build, which leads to:

```
The Broccoli Plugin: [BroccoliMergeTrees: Assembler (vendor & appJS)] failed with:
Error: ENOENT: no such file or directory, open '/Users/tbieniek/Code/foo/tmp/source_map_concat-input_base_path-URGBqt8l.tmp/node_modules/animation-frame/AnimationFrame.js'
    at Object.fs.openSync (fs.js:653:18)
    at Object.fs.readFileSync (fs.js:554:33)
    at SourceMap.addFile (/Users/tbieniek/Code/foo/node_modules/fast-sourcemap-concat/lib/source-map.js:75:31)
    at /Users/tbieniek/Code/foo/node_modules/ember-cli-tree-shaker/node_modules/broccoli-concat/concat.js:200:16
    at Array.forEach (<anonymous>)
    at Concat.<anonymous> (/Users/tbieniek/Code/foo/node_modules/ember-cli-tree-shaker/node_modules/broccoli-concat/concat.js:198:24)
    at /Users/tbieniek/Code/foo/node_modules/fast-sourcemap-concat/lib/source-map.js:419:12
    at initializePromise (/Users/tbieniek/Code/foo/node_modules/fast-sourcemap-concat/node_modules/rsvp/dist/rsvp.js:567:5)
    at new Promise (/Users/tbieniek/Code/foo/node_modules/fast-sourcemap-concat/node_modules/rsvp/dist/rsvp.js:1039:33)
    at SourceMap.end (/Users/tbieniek/Code/foo/node_modules/fast-sourcemap-concat/lib/source-map.js:406:10)
```

**Comment by kellyselden** [comment]
I think I know what the issue is. We need to add the node_modules tree to the ignored list. Only vendor and bower_components are there now.

**Comment by kellyselden** [comment]
Should be fixed in ember-cli-dependency-graph v0.1.2. Mind giving it another go?

**Comment by Turbo87** [comment]
yep, seems to work now
